integration-tests/src/test/java/org/zowe/apiml/integration/authentication/providers/QueryTest.java

```diff
@@ -39,26 +40,21 @@
     private final static String SCHEME = ConfigReader.environmentConfiguration().getGatewayServiceConfiguration().getScheme();
     private final static String HOST = ConfigReader.environmentConfiguration().getGatewayServiceConfiguration().getHost();
     private final static int PORT = ConfigReader.environmentConfiguration().getGatewayServiceConfiguration().getPort();
-    private final static String BASE_PATH = "/gateway/api/v1";
-    private final static String BASE_PATH_OLD_FORMAT = "/api/v1/gateway";
-    private final static String QUERY_ENDPOINT = "/auth/query";
     private final static String PASSWORD = ConfigReader.environmentConfiguration().getCredentials().getPassword();
     private final static String USERNAME = ConfigReader.environmentConfiguration().getCredentials().getUser();
     private final static String COOKIE = "apimlAuthenticationToken";
 
-    public static final String QUERY_ENDPOINT_URL = String.format("%s://%s:%d%s%s", SCHEME, HOST, PORT, BASE_PATH, QUERY_ENDPOINT);
-    public static final String QUERY_ENDPOINT_URL_OLD_FORMAT = String.format("%s://%s:%d%s%s", SCHEME, HOST, PORT, BASE_PATH_OLD_FORMAT, QUERY_ENDPOINT);
-
+    public static final String QUERY_ENDPOINT_URL = String.format("%s://%s:%d%s", SCHEME, HOST, PORT, ROUTED_QUERY);
+    
     private String token;
 
     private static String[] queryUrlsSource() {
-        return new String[]{QUERY_ENDPOINT_URL, QUERY_ENDPOINT_URL_OLD_FORMAT};
+        return new String[]{QUERY_ENDPOINT_URL};
```

Comment:
Does it make sense to have it parametrized?

Reply:
Removed